content/resources/courses.md

```diff
@@ -25,4 +25,5 @@ meta:
 # Courses
 
 - [Learn Vue by Building and Deploying a CRUD App](https://testdriven.io/courses/learn-vue/) - This course is focused on teaching the fundamentals of Vue by building and testing a web application using Test-Driven Development (TDD)
+- [Introduction to Vue.js](https://frontendmasters.com/courses/vue/) - An introductory course that explores the basics of features like directivesand modifiers, and concepts lke conponents, reactive programming, Nuxt.js, and Vuex
```

Comment:
```suggestion
- [Introduction to Vue.js](https://frontendmasters.com/courses/vue/) - An introductory course that explores the basics of features like directives and modifiers, and concepts like components, reactive programming, Nuxt.js, and Vuex
```

Reply:
Yeah @rmjordas. The course was very useful for me and a lot of people I know who write Vue.js I'll fix the typos right away.